src/platform/silabs/wifi/WifiInterface.h

```diff
@@ -177,6 +177,16 @@ extern WfxRsi_t wfx_rsi;
 
 /* Updated functions */
 
+/**
+ * @brief Function initalize the WiFi module before starting WiFi task.
+ *
+ * @return sl_status_t SL_STATUS_OK, if the initialization succeeded
+ *                     SL_STATUS_ALLOCATION_FAILED, if there are a memory allocation failure,
+ *                     SL_STATUS_FAILURE, otherwise
+ */
+
+sl_status_t InitWiFiStack(void);
```

Comment:
Public APIs in the Wi-Fi interface should always return CHIP_ERROR. The error swap must be done inside the function.